homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -465,6 +701,17 @@ def source(self) -> str | None:
 
         return self._source_change.name
 
+    @property
+    def extra_state_attributes(self) -> dict[str, Any] | None:
+        """Return information that is not returned anywhere else."""
+        attributes: dict[str, Any] = {}
+
+        # Add Beolink attributes
+        if self._beolink_attribute:
+            attributes.update(self._beolink_attribute)
```

Comment:
Is all this data really useful as state attributes, or should some or all of it instead be returned by a service? How do you expect users to make use of these state attributes?

Reply:
The attributes could be returned by services, but I think that would complicate overview of the whole situation, which of course is only relevant if the user has many Beolink compatible devices.